prefect_gcp/aiplatform.py

```diff
@@ -219,8 +219,10 @@ def _build_job_spec(self) -> "CustomJobSpec":
         )
         if service_account is None:
             raise ValueError(
-                "Could not detect a service_account through gcp_credentials; "
-                "please provide a service_account in VertexAICustomTrainingJob"
+                "A service account is required for the Vertex job. "
+                "A service account could not be detected in the attached credentials; "
```

Comment:
```suggestion
                "A service account could not be detected in the attached gcp_credentials; "
```

Reply:
I think saying `gcp_credentials` was part of the cause of this confusion. 

1. The field has a default, the user might not set this field or know anything about.
2. Even if the field is set by the users, the credentials can be loaded from the environment.

I think it's simply clearer to mention that the credentials that this object has access to (however they got there) does not include a service account.

Reply:
@desertaxle can you take a look at this?